examples/src/main/java/io/kubernetes/client/examples/Example.java

```diff
@@ -9,36 +9,230 @@
+    For API_SERVER_NAME, you can get the server name as follows.
+    $ kubectl cluster-info|grep master
+    Kubernetes master is running at https://*****************.hcp.japaneast.azmk8s.io:443
+     */
+    private final static String API_SERVER_NAME = "https://*****************.hcp.japaneast.azmk8s.io";
+    /*
+    For ACCESS_TOKEN , you can get the token as follows
+    $ kubectl describe secret $(kubectl get secrets | grep default | cut -f1 -d ' ') | grep -E '^token' | cut -f2 -d':' | tr -d '\t'
+     */
+    private static final String ACCESS_TOKEN = "********************************";
+    
+    private final static Logger LOGGER = Logger.getLogger(Example.class.getName());
+
+    /**
+     * Constructor
+     * @throws java.io.IOException
+     */
+    public Example() throws IOException {
+        ApiClient client = Config.defaultClient();
+        //If you want to use specific k8s cluster and access token, please use following?
+        //ApiClient client = Config.fromToken(API_SERVER_NAME, ACCESS_TOKEN, false);
+        Configuration.setDefaultApiClient(client);
+        corev1Api = new CoreV1Api(client);
+    }
+
+    /**
+     * Main method
+     *
+     * @param args
+     */
+    public static void main(String[] args) {
+        try {
+            Example operation = new Example();
+            operation.executeCommand();
+        } catch (ApiException | IOException ex) {
+            Logger.getLogger(Example.class.getName()).log(Level.SEVERE, null, ex);
+        }
+    }
+
+    public void executeCommand() throws ApiException {
+
+        //ScaleUp/ScaleDown the Deployment pod
+        System.out.println("----- Scale Deployment Start -----");
+        scaleDeployment("account-service", 5);
+        System.out.println("----- Scale Deployment End -----");
+
+        //List all of the namaspaces and pods
+        List<String> nameSpaces = getAllNameSapces();
+        nameSpaces.stream().forEach(namespace -> {
+            try {
+                System.out.println("----- " + namespace + " -----");
+                getNamespacedPod(namespace).stream().forEach(System.out::println);
+            } catch (ApiException ex) {
+                LOGGER.log(Level.SEVERE, null, ex);
+            }
+        });
+
+        //Print all of the Services
+        System.out.println("----- Print list all Services Start -----");
+        List<String> services = getServices();
+        services.stream().forEach(System.out::println);
+        System.out.println("----- Print list all Services End -----");
+
+        //Print log of specific pod
+        System.out.println("----- Print Log of Specific Pod Start -----");
+        printLog(DEFAULT_NAME_SPACE, "account-service-cbd44cc8-tq5hb");
+        System.out.println("----- Print Log of Specific Pod End -----");
+    }
+
+    /**
+     * Get all namespaces in k8s cluster
+     *
+     * @return
+     * @throws ApiException
+     */
+    public List<String> getAllNameSapces() throws ApiException {
```

Comment:
typo, this should be `getAllNamespaces`

Reply:
Yes, I will modify it !! Thank you so much !!